include/sketch.h

```diff
@@ -4,13 +4,11 @@
 #include <iostream>
 #include <vector>
 #include "bucket.h"
-#include "prime_generator.h"
 #include "types.h"
-#include "update.h"
 #include "util.h"
 #include <gtest/gtest_prod.h>
 
-#define bucket_gen(x) double_to_ull(log2(x)+1)
+#define bucket_gen(x, c) double_to_ull((c)*(log2(x)+1))
```

Comment:
I'm not sure this is the way we want to do this. We want to convert `log2(x)` to an integer, then multiply that by `c` right?

Reply:
I don't think the order of the conversion matters that much. We're toying with c=.5 and other non-integer factors, so we may as well do just the integer conversion afterwards.

Reply:
Alright, sounds good